x-pack/plugins/security_solution/public/resolver/view/graph_controls.tsx

```diff
@@ -64,7 +66,7 @@ const GraphControlsComponent = React.memo(
     className?: string;
   }) => {
     const dispatch: (action: ResolverAction) => unknown = useDispatch();
-    const scalingFactor = useSelector(selectors.scalingFactor);
+    const scalingFactor = useShallowEqualSelector(selectors.scalingFactor);
```

Comment:
The resolver code expects all comparisons to be done strictly. Please do not introduce these changes without a justification.